include/deal.II/fe/fe_values.h

```diff
@@ -2026,7 +2026,7 @@ class FEValuesBase :
    *
    * @post <code>laplacians[q]</code> will contain the Laplacian of the field
    * described by fe_function at the $q$th quadrature point.
-   * <code>gradients[q][i][j]</code> represents the $(i,j)$th component of the
+   * <code>laplacians[q][i][j]</code> represents the $(i,j)$th component of the
    * matrix of second derivatives at quadrature point $q$.
```

Comment:
`laplacians[q][i][j]` seems wrong. The Laplacians are a scalar for a scalar element evaluation as is done here, so one should not talk about i and j here. How about saying `laplacians[q]` represents the sum of the diagonal entries of the Hessian matrix of second derivatives at quadrature point $q$?